test/integration/user_can_see_past_orders_test.rb

```diff
@@ -37,24 +38,62 @@ class UserCanSeePastOrdersTest < ActionDispatch::IntegrationTest
   end
 
   test "user can place an order and is redirected to the order history page" do
-    create_and_login_user
-    user = User.first
-    add_items_to_cart(2)
-    visit "/cart"
-    click_button "Checkout"
+    checkout_user(2)
 
     assert page.has_content?("Order History")
 
-    within(".cart-table") do # within() vs. within_table() ?!
+    within(".cart-table") do
       assert page.has_content?("Trips Ordered")
       assert page.has_content?("Total Price")
       assert page.has_content?("Date Ordered")
 
-      # assert find('tr', text: "Trips Ordered").has_content?("Hiking")
       assert page.has_content?("Hiking the Alps 1 (Travellers: 1)")
       assert page.has_content?("Hiking the Alps 1 (Travellers: 2)")
       assert page.has_content?("$3,003")
       assert page.has_content?("#{Order.first.created_at.strftime("%B %d, %Y")}")
     end
   end
+
+  test "authenticated user can see individual past orders" do
+    checkout_user(2)
+    order = Order.first
+    order_timestamp = order.created_at
+    formatted_timestamp = "#{order_timestamp.strftime("%B %d, %Y")} at #{order_timestamp.strftime("%H:%M")}"
```

Comment:
Line is too long. [108/80]<br>Prefer single-quoted strings inside interpolations.